static/app/views/organizationCreate/index.tsx

```diff
@@ -9,9 +12,32 @@ import ConfigStore from 'sentry/stores/configStore';
 import {OrganizationSummary} from 'sentry/types';
 import {normalizeUrl} from 'sentry/utils/withDomainRequired';
 
+enum RegionDisplayName {
+  US = '🇺🇸 United States of America (USA)',
+  DE = '🇪🇺 European Union (EU)',
+}
+
+function get_region_choices() {
```

Comment:
Most typescript code is using camelBacked casing eg. `getRegionChoices()`

Reply:
oops, python brain is still kicking in 🙈